src/app/AttributeAccessInterface.h

```diff
@@ -97,10 +99,19 @@ class AttributeValueEncoder
     public:
         ListEncodeHelper(AttributeValueEncoder & encoder) : mAttributeValueEncoder(encoder) {}
 
-        template <typename... Ts>
-        CHIP_ERROR Encode(Ts &&... aArgs) const
+        template <typename T, std::enable_if_t<DataModel::IsFabricScoped<T>::value, bool> = true>
+        CHIP_ERROR Encode(T && aArg) const
         {
-            return mAttributeValueEncoder.EncodeListItem(std::forward<Ts>(aArgs)...);
+            // If the fabric index does not match the one requests, just skip this item.
+            ChipLogError(DataManagement, "asdjasdjkasdasdjsakdasl");
```

Comment:
The log message seems odd here...guessing debug?

Reply:
Yes... Forgot to remove this after fixing the issue during coding.